client/dashboard/plugins/scheduled-updates/new/components/sites-selection/index.tsx

```diff
@@ -0,0 +1,96 @@
+import { DataViews, Field, View, filterSortAndPaginate, type Action } from '@wordpress/dataviews';
+import { __ } from '@wordpress/i18n';
+import { useMemo, useState } from 'react';
+import { DataViewsCard } from '../../../../../components/dataviews-card';
+import { Name, URL, SiteIconLink } from '../../../../../sites/site-fields';
+import { getSiteDisplayName } from '../../../../../utils/site-name';
+import { getSiteDisplayUrl } from '../../../../../utils/site-url';
+import { useEligibleSites } from '../../../hooks/use-eligible-sites';
+import type { Site } from '@automattic/api-core';
+
+const siteFields: Field< Site >[] = [
+	{
+		id: 'name',
+		label: __( 'Site' ),
+		enableGlobalSearch: true,
+		getValue: ( { item } ) => getSiteDisplayName( item ),
+		render: ( { field, item } ) => <Name site={ item } value={ field.getValue( { item } ) } />,
+	},
+	{
+		id: 'url',
+		label: __( 'URL' ),
+		enableGlobalSearch: true,
+		getValue: ( { item } ) => getSiteDisplayUrl( item ),
+		render: ( { field, item } ) => <URL site={ item } value={ field.getValue( { item } ) } />,
+	},
+	{
+		id: 'icon.ico',
+		label: __( 'Site icon' ),
+		render: ( { item } ) => <SiteIconLink site={ item } />,
```

Comment:
Also not sure if we should be rendering the name and URL as links, but it does keep the view (expectation?) consistent/aligned with the sites' page.

<img width="500" alt="Screenshot 2025-09-09 at 5 33 40 PM" src="https://github.com/user-attachments/assets/a7cdc338-d6e3-4a7f-92fb-ff092f94baf0" />